src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Ui;
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+/**
+ * Represents a command to exit Duke.
+ *
+ * @author Derrick Khoo
+ */
+public class ByeCommand extends Command {
+
+    /**
+     * Returns true if command was to exit Duke.
+     * @return true
+     */
+    @Override
+    public boolean isDone() {
+        return true;
+    }
+
+    /**
+     * Handles the command to exit Duke.
+     * @param storage  the <code>Storage</code> dealing with loading and saving tasks in a file
+     * @param ui       the user interfaces that deals with user inputs
+     * @param taskList the list of tasks
+     * @throws DukeException if there is an error saving to the file
+     */
+
```

Comment:
Perhaps a line between the method description and parameters would better follow the coding standards here? I believe this could apply to other similar parts of your code.

Referencing the comments section in - https://se-education.org/guides/conventions/java/intermediate.html